NOTICE.md

```diff
@@ -28,7 +28,7 @@ SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 
 ## Source Code
 
-The project maintains the following source code repositories:
+The project maintains the following source code repositories. An up to date list can be also found in the [Eclipse GlassFish project page]([url](https://projects.eclipse.org/projects/ee4j.glassfish/developer)).
```

Comment:
This url does not seem to be rendered as link (https://github.com/eclipse-ee4j/glassfish/blob/d315e6f332527b61c7d6e02b7019a5b945ad32da/NOTICE.md).

![image](https://github.com/user-attachments/assets/945a52e5-cb97-4a1b-8893-c774a92499f1)

Is that as expected?

Reply:
Thanks. It was a typo, fix now.